src/kbc_modules/sample_kbc/mod.rs

```diff
@@ -36,18 +41,14 @@ impl KbcInterface for SampleKbc {
     }
 
     fn decrypt_payload(&mut self, annotation: &str) -> Result<Vec<u8>> {
-        // Parse the annotation, and obtain the access information of KBS and the field content to be decrypted.
         let annotation_packet: AnnotationPacket = serde_json::from_str(annotation)?;
-        self.encrypted_payload = annotation_packet.wrapped_key;
-
-        let cipher_text: &Vec<u8> = &self.encrypted_payload;
-        let decrypting_key = Key::from_slice(b"passphrasewhichneedstobe32bytes!");
-        let cipher = Aes256Gcm::new(decrypting_key);
-        let nonce = Nonce::from_slice(b"unique nonce");
+        self.encrypted_payload = annotation_packet.wrapped_data;
```

Comment:
what is the point of the encrypted_payload field?

Reply:
@fitzthum I have tested the following contents: encrypt a container image using skopeo and sample KBS, and decrypt the container image using skopeo and AA sample KBC. This test can run successfully.

The encrypted_payload field stores the [PrivateLayerBlockCipherOptions object](https://github.com/containers/ocicrypt/blob/main/blockcipher/blockcipher.go#L34) encrypted by KEK, which contains the necessary information and symmetric key for encryption. This field comes from the annotation of the container image layer.